tests/vars.sample.yaml

```diff
@@ -29,3 +29,7 @@ timesync_ntp_servers:
 
 # Auth URL to use for adopted Keystone.
 auth_url: http://keystone-public-openstack.apps-crc.testing
+
+# Whether source env is OSPD Director Operator
+# i.e. https://github.com/openstack-k8s-operators/osp-director-operator
+ospdo_src: false
```

Comment:
theres a couple of those it means you need to add a newline

Reply:
ack